src/main/java/food/finder/foodfinder/user/User.java

```diff
@@ -8,6 +8,7 @@
 import org.springframework.data.neo4j.core.schema.Property;
 import org.springframework.data.neo4j.core.schema.Relationship;
 
+import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
 
```

Comment:
Apparently, you can't line comment on something that doesn't have a diff ...

lombok is really cool, so all the fields in this class now have get and set methods. is there any downside to using lombok in a project?

Reply:
The only downside that I can think of is having more methods are are used but that happens at compile time so doesn't effect the readability. There are some caveats using Lombok in combination with other libraries like hibernates but they are really specific cases.